.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
What is all this stuff in the `.vs` folder? Doesn't look like it should be here.